index.html

```diff
@@ -40,21 +40,21 @@ <h1 class="down_word">I’m a software developer</h1>
             </section>
             <section class="social_links">
                 <ul class="social_links">
-                    <li>  <a href="https://twitter.com/PatientMusafir1"><img src="icon/twitter.png" alt=""  class="icon"> </a></li>
-                    <li>  <a  href="https://www.linkedin.com/in/ishimwe-joseph-patient-0537b4155/"><img src="icon/Linkedin icon.png" alt=""  class="icon"> </a></li>
-                    <li> <a  href="#">  <img src="icon/medium.png" alt=""  class="icon">  </a></li>
-                    <li> <a  href="https://github.com/ishimwepati"> <img src="icon/github.png" alt=""  class="icon"> </a></li>
-                    <li> <a href="#"> <img src="icon/angellist.png" alt=""  class="icon"> </a></li>
+                    <li>  <a href="https://twitter.com/PatientMusafir1"><img src="icon/twitter.png" alt="Twitter Logo with a link to twitter"  class="icon"> </a></li>
+                    <li>  <a  href="https://www.linkedin.com/in/ishimwe-joseph-patient-0537b4155/"><img src="icon/Linkedin icon.png" alt="Linkedin Logo with a link to Linkedin"  class="icon"> </a></li>
+                    <li> <a  href="#">  <img src="icon/medium.png" alt="Medium Logo with a link to Medium"  class="icon">  </a></li>
+                    <li> <a  href="https://github.com/ishimwepati"> <img src="icon/github.png" alt="Github Logo with a link to Github"  class="icon"> </a></li>
+                    <li> <a href="#"> <img src="icon/angellist.png" alt="Angellist Logo with a link to angellist"  class="icon"> </a></li>
                     <br>
                 </ul>
                 <br>
             </section>
-            <button class="disable_button"><img src="icon/Disabled.png"></button>
+            <button class="disable_button"><img src="icon/Disabled.png" alt="Disabled button"></button>
         </section>
     </section>
     <section class="work">
         <h1 class="upper_word"><br>My Recent <br> Works</h1>
```

Comment:
- [ ] Kindly make sure your heading levels have a meaningful hierarchy. Ideally the page starts with an `h1` tag which is usually used only once in a page.
### Your portfolio heading hierarchy from the "About me" section
![Screenshot from 2023-06-21 09-12-22](https://github.com/ishimwepati/Portfolio/assets/91142432/06ac5e0d-2436-486a-9fcc-e5513cfc0c88)